locale/en_US.json

```diff
@@ -40,18 +40,18 @@
     "command.connect.error.already_in_channel.continue": "Continue",
     "command.connect.error.already_in_channel.cancel": "Canceled",
     "command.disconnect.name": "disconnect",
-    "command.disconnect.description": "Disconnect from channel",
```

Comment:
Are you sure about this? This is the description of disconnect command
![image](https://user-images.githubusercontent.com/56459446/233831541-33d60ee5-3725-4cfc-882d-9a881010d60e.png)

Reply:
ah ok